copter/source/docs/tuning-process-instructions.rst

```diff
@@ -9,26 +9,138 @@ Setting the aircraft up ready for tuning
+                xy=(5, 0.55), xycoords='data',
+                xytext=(70, -2), textcoords='offset points',
+                bbox=dict(boxstyle="round", fc="w"),
+                arrowprops=dict(arrowstyle="->"))
+    ax.annotate('10 inches props',
+                xy=(10, 0.65), xycoords='data',
+                xytext=(-70, 30), textcoords='offset points',
+                bbox=dict(boxstyle="round", fc="w"),
+                arrowprops=dict(arrowstyle="->"))
+    ax.annotate('20 inches props',
+                xy=(20, 0.75), xycoords='data',
+                xytext=(-70, -50), textcoords='offset points',
+                bbox=dict(boxstyle="round", fc="w"),
+                arrowprops=dict(arrowstyle="->"))
+    ax.set_xlabel('Props size in inches')
+    ax.set_ylabel('MOT_THST_EXPO value')
+    ax.set_title('Theorical relationship between MOT_THST_EXPO value \n and props size in inches')
+
+Controler setup
+^^^^^^^^^^^^^^^
+- :ref:`INS_ACCEL_FILTER <INS_ACCEL_FILTER>` -  10Hz to 20Hz  # todo explain those value ?
+- :ref:`INS_GYRO_FILTER <INS_GYRO_FILTER>` - 80Hz for 5 inch props, 40Hz for 10 inch props, 20Hz for 20 inch props
+- :ref:`ATC_RAT_PIT_FILT <ATC_RAT_PIT_FILT__AC_AttitudeControl_Multi>` - :ref:`INS_GYRO_FILTER <INS_GYRO_FILTER>` / 2
+- :ref:`ATC_RAT_RLL_FILT <ATC_RAT_RLL_FILT__AC_AttitudeControl_Multi>` - :ref:`INS_GYRO_FILTER <INS_GYRO_FILTER>` / 2
+- :ref:`ATC_RAT_YAW_FILT <ATC_RAT_YAW_FILT__AC_AttitudeControl_Multi>` - 2
+- :ref:`ATC_ACCEL_P_MAX <ATC_ACCEL_P_MAX>` - 110000 for 10 inch props, 50000 for 20 inch props, 20000 for 30 inch props
+- :ref:`ATC_ACCEL_R_MAX <ATC_ACCEL_R_MAX>` - 110000 for 10 inch props, 50000 for 20 inch props, 20000 for 30 inch props
+- :ref:`ATC_ACCEL_Y_MAX <ATC_ACCEL_Y_MAX>` - 27000 for 10 inch props, 18000 for 20 inch props, 9000 for 30 inch props
+- :ref:`ACRO_YAW_P <ACRO_YAW_P>` - 0.5 x :ref:`ATC_ACCEL_Y_MAX <ATC_ACCEL_Y_MAX>` / 4500
+
+.. plot::
+
+    import matplotlib.pyplot as plt
+    fig, ax = plt.subplots()
+    ax.plot([5, 10, 20], [80, 40, 20])
+    ax.annotate('5 inches props',
+                xy=(5, 80), xycoords='data',
+                xytext=(70, -2), textcoords='offset points',
+                bbox=dict(boxstyle="round", fc="w"),
+                arrowprops=dict(arrowstyle="->"))
+    ax.annotate('10 inches props',
+                xy=(10, 40), xycoords='data',
+                xytext=(-100, -2), textcoords='offset points',
+                bbox=dict(boxstyle="round", fc="w"),
+                arrowprops=dict(arrowstyle="->"))
+    ax.annotate('20 inches props',
+                xy=(20, 20), xycoords='data',
+                xytext=(-50, 50), textcoords='offset points',
+                bbox=dict(boxstyle="round", fc="w"),
+                arrowprops=dict(arrowstyle="->"))
+    ax.set_xlabel('Props size in inches')
+    ax.set_ylabel('INS_GYRO_FILTER value')
+    ax.set_title('Theorical relationship between INS_GYRO_FILTER value \n and props size in inches')
+
```

Comment:
Approximate relationship between INS_GYRO_FILTER value \n and props size in inches